### DISCUSSION TITLE
Interconnecting budibase apps

### DISCUSSION THREAD

**Discussion by thomaskamalakis** [Budibase/budibase#5337] [Help] [answered]
Congrats on making an excellent framework for building apps "using the mouse"

So quick question for a newbie like me: how do you connect budibasedB table X of app Y to app Z? Can we use the API to achieve this?

CouchdB as a data source has no credentials options...

**Comment by melohagan** [comment, marked as answer]
Hi @thomaskamalakis 

You can use the REST connector (https://docs.budibase.com/docs/rest) in combination with the budibase API (https://docs.budibase.com/reference/post_applications)
